cfpb/consumerfinance.gov
# Revert migration squashing in #7877

This reverts commit 1084fe74106895be9a27babba67ff289e7dec888, reversing changes made to b2f890d4508ed539b34c9eb3a5dd014599015a9e.

I screwed up the migration squashing in #7877, and introduced a circular migration dependency. Happily since we never removed the old migration files from the environment we can just rollback the previous commit.

## How to test this PR

To test this, after checking out this branch you can refresh your database do a `cfgov/manage.py showmigrations` and see the old migration names there. `tox -e validate-migrations` works as well.

## Checklist
